Makefile

```diff
@@ -33,13 +33,23 @@ build-image: $(STAMP_FILE) ## 🐳 Build the Docker image
 run-image: build-image ## 🏃 Run the Docker image
 	docker run --rm -p 127.0.0.1:8545:8545 ethrex --http.addr 0.0.0.0
 
-$(SPECTEST_ARTIFACT):
-	rm -f tests_*.tar.gz # Delete older versions
-	curl -L -o $(SPECTEST_ARTIFACT) "https://github.com/ethereum/execution-spec-tests/releases/download/$(SPECTEST_VERSION)/fixtures_stable.tar.gz"
+# This target is used to store the current version of EF tests downloaded.
+# If the SPECTEST_LINK variable is modified, this target will be run and the file modified, which will trigger a new $(SPECTEST_ARTIFACT) build
+# Thanks to:
+# - https://www.technovelty.org/tips/the-stamp-idiom-with-make.html
+# - https://stackoverflow.com/questions/27791578/makefile-how-to-detect-changes-within-the-makefile-itself
+EF_TEST_STAMP_FILE := .ef_tests_stamp_file
```

Comment:
you shouldn't have to use a stamp file in this case, previously there was no stamp file either. Im just proposing that you keep everything similar like it was before, but updating the versions.

Reply:
<img width="1146" alt="Screenshot 2025-02-03 at 20 45 20" src="https://github.com/user-attachments/assets/7ac00f8b-b1d5-498d-b9fa-a9e224391144" />